mod_rewrite/.htaccess

```diff
@@ -115,16 +115,20 @@ RewriteRule "^examples/jupyter/input/ck2yaml_demo.ipynb.html" "stable/userguide/
 
 RewriteRule "^examples/jupyter/reactors/1D_packbed.ipynb.html" "stable/examples/python/reactors/1D_packed_bed.html" [R=permanent,L]
 RewriteRule "^examples/jupyter/reactors/1D_pfr_surfchem.ipynb.html" "stable/examples/python/reactors/1D_pfr_surfchem.html" [R=permanent,L]
-RewriteRule "^examples/jupyter/reactors/batch_reactor_ignition_delay_NTC.ipynb.html" "stable/examples/python/reactors/NonIdealShockTube.html" [R=permanent,L]
+RewriteRule "^examples/jupyter/reactors/batch_reactor_ignition_delay_NTC.ipynb.html" "stable/examples/python/reactors/non_ideal_shock_tube.html" [R=permanent,L]
 RewriteRule "^examples/jupyter/reactors/interactive_path_diagram.ipynb.html" "stable/examples/python/kinetics/interactive_path_diagram.html" [R=permanent,L]
-RewriteRule "^examples/jupyter/reactors/nonideal_shock_tube.ipynb.html" "stable/examples/python/reactors/NonIdealShockTube.html" [R=permanent,L]
+RewriteRule "^examples/jupyter/reactors/nonideal_shock_tube.ipynb.html" "stable/examples/python/reactors/non_ideal_shock_tube.html" [R=permanent,L]
```

Comment:
I guess we still have website updates that can't be merged until the next release. "stable" currently points to the 3.1 docs, where these have the IntialCaps names.